src/components/tooltip/tooltip-test.stories.tsx

```diff
@@ -150,7 +150,10 @@ const SecondaryButton = forwardRef<HTMLButtonElement, ButtonProps>(
 );
 SecondaryButton.displayName = "Tooltip";
 
-export const TooltipComponent = ({ message, ...props }: TooltipProps) => (
+export const TooltipComponent = ({
+  message,
+  ...props
+}: Partial<TooltipProps>) => (
```

Comment:
can it be as it was without `Partial`?

Reply:
Hi @DlgSHi , I'm getting type error after removing "Partial" from Tooltip component.